SwiftyContextMenu/ContextMenuBlurView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ContextMenuBlurView.swift
+//  SwiftyContextMenu
+//
+//  Created by Paul Bancarel on 28/11/2020.
+//
+
+class ContextMenuBlurView: IntensityVisualEffectView {
+    
+    internal class var intensity: CGFloat { 1.0 }
```

Comment:
I don't think internal is required since it is the default access control qualifier